spec/unit/type/yumrepo_spec.rb

```diff
@@ -399,5 +399,22 @@
       it_behaves_like "a yumrepo parameter that can be absent", :deltarpm_metadata_percentage
       it_behaves_like "a yumrepo parameter that expects a natural value", :deltarpm_metadata_percentage
     end
+
+    describe "username" do
+      it_behaves_like "a yumrepo parameter that can be absent", :username
+    end
+
+    describe "password" do
+      it_behaves_like "a yumrepo parameter that can be absent", :password
+      it "redacts password information from the logs" do
```

Comment:
The CDO in me says there should be a space between the it_behaves_like and the it 
(shrug)